README.md

```diff
@@ -28,6 +28,22 @@ To generate the file that will become https://github.com/docker-library/official
 # commit your changes first!
 ./populate.sh library
 ```
+# Running
+
+When running the Varnish image, a `varnishd` process will be started that listens on the following ports:
+
+* port `80` for *plain HTTP*
+* port `8443` for the *PROXY protocol*
```

Comment:
you can just add "see the TLS section" and drop l38, I don't know of anyone actually using the PROXY protocol for the load-balancer, the main use here is really TLS termination. You can bounce the proxy link to the TLS section